src/manifest.c

```diff
@@ -594,13 +594,17 @@ struct manifest *load_mom(int version, bool latest, bool mix_exists, int *err)
 	/* Make a copy of the Manifest for the completion code */
 	if (latest) {
 		char *momcopy;
-		rm_bundle_file(".MoM");
-		string_or_die(&momcopy, "/bin/cp \"%s\" \"%s/%s/.MoM\" 2>/dev/null",
-			      filename, path_prefix, BUNDLES_DIR);
+		char *momdir;
+		string_or_die(&momdir, "%s/%s", path_prefix, MOM_DIR);
+		swupd_rm(momdir);
```

Comment:
We don't need to remove the directory. this is going to fail if there's something else in there. Can't we just remove the .MoM file?

Reply:
actually it won't fail because swupd_rm will delete the directory and its content. The .MoM file is the only file in that directory at the moment. Having said that it is probably better to only remove the file as you suggests in case we want to add more stuff to that directory in the future. I'll make the change

Reply:
Oh, I didn't know that. Nice. But anyway it's better to delete just the file.